cleanlab/internal/label_quality_utils.py

```diff
@@ -96,17 +96,17 @@ def get_normalized_entropy(pred_probs: np.ndarray, min_allowed_prob: float = 1e-
 
     Parameters
     ----------
-    pred_probs : np.ndarray (shape (N, K))
+    pred_probs
       Each row of this matrix corresponds to an example x and contains the model-predicted
       probabilities that x belongs to each possible class: P(label=k|x)
 
-    min_allowed_prob : float, default=1e-6
+    min_allowed_prob
```

Comment:
Note this still doesn't exactly match our desired arg format which has been implemented in the token_classification module, cf:  https://github.com/cleanlab/cleanlab/blob/master/cleanlab/token_classification/summary.py#L45

This should be:
```
pred_probs:
  Each row ...
```